kolibri/content/utils/sqlalchemybridge.py

```diff
@@ -0,0 +1,126 @@
+import os
```

Comment:
The comments that *are* in this file are very useful, and... more of them could be helpful! (Largely due to unfamiliarity on my side...)

Maybe just some one-liner docstrings on the less obvious functions, e.g. `get_base`?